src/pages/[language]/blog/[slug]/index.vue

```diff
@@ -202,17 +205,22 @@
       : body
   );
 
-  const tocItems = computed(() => {
+  const items = computed(() => {
     return data.value.page.items
-      .filter(item => item.title)
-      .map(({ title }) => {
-        return {
-          slug: slugify(title),
-          title
+      .map((item) => {
+        return item.title ? {
+          slug: slugify(item.title),
```

Comment:
Could we call `slug` `titleId` here? I understand `slug` describes that it's the slugified title, but I would prefer to call the property what it's used for.